frontend/src/pages/WorkspaceSettings/WorkspaceSettings.tsx

```diff
@@ -2,7 +2,7 @@ import Alert from '@components/Alert/Alert'
 import { FieldsBox } from '@components/FieldsBox/FieldsBox'
 import { AdminRole } from '@graph/schemas'
 import { Box } from '@highlight-run/ui/components'
-import AutoJoinForm from '@pages/WorkspaceTeam/components/AutoJoinForm'
+import { AutoJoinForm } from '@pages/WorkspaceTeam/components/AutoJoinForm'
```

Comment:
does it make sense to have the auto join checkbox on the settings page? seems redundant to me with having the email inputs, since if i want to turn it off, i can just remove all the email domains from the list.

Reply:
Synced with Julian here:

> The reason we put the check-box there is to easily opt-out of this. I believe that this won’t be unchecked/checked quite frequently, therefor I don’t think it will be confusing. In the unchecked state we could maybe show the text that we have in the callout on the sign up process. The checked state could show the input instead

Personally, I like always having the select showing, but don't feel too strongly about the checkbox if we have an easy way to clear the select. The checkbox also adds some context, so want to callout that if its removed we should replace with some text describing what the select input controls.